composer.json

```diff
@@ -28,32 +30,37 @@
     }
   ],
   "require": {
-    "php": "^7.4 || ^8.0",
-    "symfony/config": ">= 5.3 <6.0",
-    "symfony/dependency-injection": ">= 5.3 <6.0",
-    "symfony/http-kernel": ">= 5.3 <6.0",
-    "symfony/framework-bundle": ">= 5.3 <6.0",
-    "doctrine/annotations": "^1.6",
-    "nelmio/api-doc-bundle": "^3.2",
+    "php": "^8.0",
+    "symfony/config": "^5.3 || ^6.0",
+    "symfony/dependency-injection": "^5.3 || ^6.0",
+    "symfony/http-kernel": "^5.3 || ^6.0",
+    "symfony/framework-bundle": "^5.3 || ^6.0",
+    "doctrine/annotations": "^1.13",
+    "nelmio/api-doc-bundle": "^3.10",
     "sensio/framework-extra-bundle": "^5.1 || ^6.0",
-    "check24/apitk-common-bundle": "^2.2 || ^3.0",
-    "check24/apitk-header-bundle": "^2.2 || ^3.0"
+    "check24/apitk-common-bundle": "dev-feature/symfony-6 as 4.0.0",
+    "check24/apitk-header-bundle": "dev-feature/symfony-6 as 4.0.0"
```

Comment:
Both VCS-URLs need to be changed back as soon as their respective PRs are tagged and released


- [ ] https://github.com/byWulf/apitk-common-bundle/pull/15
- [ ] https://github.com/byWulf/apitk-header-bundle/pull/10